examples/templates/single-backend-java/build.sbt

```diff
@@ -0,0 +1,36 @@
+import sbt._
+import sbt.Keys._
+
+lazy val templateJavaProject = (project in file("."))
+     //enable here the backend you want to use in your application:
+     //CloudflowAkkaStreamsApplicationPlugin, CloudflowSparkApplicationPlugin, CloudflowFlinkApplicationPlugin
+    .enablePlugins(CloudflowAkkaStreamsApplicationPlugin, ScalafmtPlugin)
+    .settings(
+      scalafmtOnCompile := true,
+      libraryDependencies ++= Seq(
+	      "ch.qos.logback"     %  "logback-classic"        % "1.2.3",
```

Comment:
Some strange formatting

Reply:
weird. It looks OK on my editor. 
![image](https://user-images.githubusercontent.com/874997/79757138-f1a99580-831b-11ea-8e7e-0fa4bf7617cf.png)